src/core/platform-configs/configs.js

```diff
@@ -0,0 +1,19 @@
+// These can be spread into more files, if these configs grow
+import { platformName } from './platform-name';
+
+const mobilePlatformsConcurrentPulls = 2;
+
+export const platformConfigs = {
+  base: {
+    maxConcurrentPushRequests: 100,
+    maxConcurrentPullRequests: 32
+  },
+
+  [platformName.nativeScript]: {
+    maxConcurrentPullRequests: mobilePlatformsConcurrentPulls
+  },
+
+  [platformName.phoneGap]: {
+    maxConcurrentPullRequests: mobilePlatformsConcurrentPulls
```

Comment:
I haven't tested on PhoneGap. I set it to the same as NativeScript, just in case. We can test it out before GA.